packages/gatsby-theme-store/src/sdk/pixel/events.ts

```diff
@@ -166,3 +165,17 @@ interface PriceTag {
   isPercentual: boolean
   value: number
 }
+
+export interface PixelItem {
+  productId: string
+  productReferenceId: Maybe<string>
+  productName: string
+  brand: string
+  categoryTree: Array<{ name: string }>
+  price: number
+  // TODO currencyCode
```

Comment:
This comment should go to production?

Reply:
Yes, it should. When we start supporting multiple currencies and etc we should send it with these events, but I thought it was better to not worry about that now.